packages/rollup-plugin-html/README.md

```diff
@@ -444,6 +465,39 @@ export default {
 
 </details>
 
+## Details for Plugin rootDir
+
+When keeping the folder structures it may mean that your `rootDir` is not your current working directory.
+Image for example all the html files are generated in to a `_site-in-html` folder.
+
+```
+.
+├── _site-in-html
+│   ├── page-a
+│   │   └── index.html
+│   └── index.html
+├── rollup.config.js
+└── package.json
+```
+
+If you now provide `input: '_site-in-html/**/*.html';` then it will keep this folder which will result in this url `https://my-domain.com/_site_in-html/`.
```

Comment:
this paragraph is not entirely clear to me yet, is there a better way we can describe it? Theres also some typos, maybe we can run it through grammarly?

Reply:
grammarly and more infoe